lua.html.markdown

```diff
@@ -377,13 +379,14 @@ local a = require('mod2')  -- Prints Hi!
 local b = require('mod2')  -- Doesn't print; a=b.
 
 -- dofile is like require without caching:
-dofile('mod2.lua')  --> Hi!
-dofile('mod2.lua')  --> Hi! (runs it again)
+dofile('./mod2.lua')  --> Hi!
+dofile('./mod2.lua')  --> Hi! (runs it again)
+-- notice: the difference between this and 'require' is that you need to specify the file path and suffix
```

Comment:
you don't need a filepath, these examples don't add one

https://www.gammon.com.au/scripts/doc.php?lua=dofile

https://stackoverflow.com/questions/49033344/dofile-and-require-does-not-work-as-what-i-expected